app/code/Magento/InventoryConfigurableProduct/Bulk.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryConfigurableProduct;
+
+use Magento\Catalog\Helper\Image;
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\Media\Config;
+use Magento\Catalog\Model\Product\Type;
+use Magento\Catalog\Model\ProductFactory;
+use Magento\Eav\Model\Entity\Attribute;
+use Magento\Framework\View\Element\Template\Context;
+
+/**
+ * Adminhtml block for fieldset of configurable product
+ *
+ * @api
+ * @since 100.0.2
+ */
+class Bulk extends \Magento\Ui\Block\Component\StepsWizard\StepAbstract
+{
+    /**
+     * @var \Magento\Catalog\Helper\Image
+     */
+    protected $image;
+
+    /**
+     * @var ProductFactory
+     */
+    private $productFactory;
+
+    /**
+     * @var Config
+     */
+    private $catalogProductMediaConfig;
+
+    /**
+     * @param Context $context
+     * @param Image $image
+     * @param Config $catalogProductMediaConfig
+     * @param ProductFactory $productFactory
+     */
+    public function __construct(
+        Context $context,
+        Image $image,
+        Config $catalogProductMediaConfig,
+        ProductFactory $productFactory
+    ) {
+        exit('ping');
+        parent::__construct($context);
+        $this->image = $image;
+        $this->productFactory = $productFactory;
+        $this->catalogProductMediaConfig = $catalogProductMediaConfig;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getCaption()
+    {
+        return __('Bulk Images &amp; Price');
+    }
+
+    /**
+     * @return string
+     */
+    public function getNoImageUrl()
+    {
+        return $this->image->getDefaultPlaceholderUrl('thumbnail');
+    }
+
+    /**
+     * Get image types data
+     *
+     * @return array
+     */
+    public function getImageTypes()
+    {
+        $imageTypes = [];
+        foreach ($this->catalogProductMediaConfig->getMediaAttributeCodes() as $attributeCode) {
+            /* @var $attribute Attribute */
+            $imageTypes[$attributeCode] = [
+                'code' => $attributeCode,
+                'value' => '',
+                'label' => $attributeCode,
+                'scope' => '',
+                'name' => $attributeCode,
+            ];
+        }
+        return $imageTypes;
+    }
+
+    /**
+     * @return array
+     */
+    public function getMediaAttributes()
+    {
+        static $simple;
```

Comment:
is that a copy-paste from legacy code, or we really want to use static variable for caching?